python-package/xgboost/core.py

```diff
@@ -1040,6 +1045,8 @@ def predict(self, data, output_margin=False, ntree_limit=0, pred_leaf=False,
             option_mask |= 0x04
         if approx_contribs:
             option_mask |= 0x08
+        if interaction_contribs:
+            option_mask |= 0x16
```

Comment:
x16 is 0b10110
Was the intent to turn on, in addition, the flags for pred_contribs and **pred_leaf**?

Reply:
Good catch. No I was thinking decimal and writing hexadecimal